CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Getting Started
+
+Okay, you have looked around the repository and found something that you want to change.
+
+Here is what you can do.
+
+## Issues
+
+It's important to check the issues to see if someone has already addressed your concern.
+
+If an issue exists
+- Comment in the issue
+
+If no issue addresses your concerns
+- Create a new issue from a template
+
+The templates are there to help fast track some information so that we can get to your issue sooner. Feel free to alter them when creating an issue if they do not match your content.
+
+## Pull Requests
+
+If you are looking to do more than just raise an issue then check to see if there are any issues labelled under `good first issue` and no one is assigned to them.
+
+These issues should be relatively easy to do and they are there specifically to let others jump in and have a go.
+
+Before you go off and make a pull request (PR) make sure to first fork the repository and work in your local repo. 
+
+Once you are ready to submit a PR make sure to mark it as a draft first until it is ready to go into the master branch.
+
+When making a pull request there are many things to consider depending on the type of request.
+
+Here are some things to consider otherwise your PR may require multiple changes.
+
+- [ ] Have you formatted your code via `cargo fmt` and `forc fmt` ?
+- [ ] What is your test coverage like?
+- [ ] Is your code clean and high quality?
+- [ ] Have you extensively documented the code itself but also added documentation regarding its usage?
```

Comment:
I think grammatically this should be "Have you not only extensively documented the code, but also added documentation regarding its usage?" or an alternative "Have you extensively documented the code and also added documentation regarding its usage?"